test/test_sub_cipher.py

```diff
@@ -0,0 +1,12 @@
+from benevolent.sub_cipher import (
+    decode_simple_sub_cipher, encode_simple_sub_cipher
+)
+
```

Comment:
We could add some more tests later to properly check that all proper errors are being thrown when needed.

Reply:
yep, later or as needed.